### PR TITLE
Revamp dbaas documentation

### PR DESCRIPTION
**Pull request by gen1us2k**

### CODE REVIEW

File: docs/dbaas/get-started.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# DBaaS
+# Getting started with DBaaS
 
 !!! caution alert alert-warning "Caution"
     DBaaS functionality is currently in [technical preview](../details/glossary.md#technical-preview) and is subject to change.
```
Comment (recharte) [inline docs/dbaas/get-started.md:4]: Should we keep this?

File: docs/dbaas/setting-up.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
-# DBaaS
+# Setting up DBaaS
 
-To enable and use the Database as a Service (DBaaS) feature in PMM, see [DBaaS](../../get-started/dbaas.html).
+To enable and use the Database as a Service (DBaaS) feature in PMM, see [DBaaS](../dbaas/get-started.html).
```
Comment (shoffman-percona) [inline docs/dbaas/setting-up.md:3]:
```suggestion
To use the Database as a Service (DBaaS) solution in PMM there are a few things that need to be setup first including a suitable Kubernetes Cluster.  If you've already got a kubernetes cluster you can jump ahead enabling it in [PMM](../dbaas/get-started.html). 
```

File: docs/dbaas/setting-up.md [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
 
-You can use [free K8s provided by Percona](https://www.percona.com/blog/private-dbaas-with-free-kubernetes-cluster/) for evaluation. 
-You can also create K8s on AWS using these [instructions](https://www.percona.com/blog/dbaas-kubernetes-in-under-20-min/). 
+You can use [free K8s provided by Percona](https://www.percona.com/blog/private-dbaas-with-free-kubernetes-cluster/) for evaluation.
```
Comment (shoffman-percona) [inline docs/dbaas/setting-up.md:5]:
```suggestion
If you don't have a Kubernetes cluster available you can use the [free K8s provided by Percona](https://www.percona.com/blog/private-dbaas-with-free-kubernetes-cluster/) for evaluation which will allow you to play around with DBaaS for 3 hours before the cluster expires.
```

File: docs/dbaas/setting-up.md [hunk 4]
```diff
@@ -1,9 +1,9 @@
...
+You can also create K8s on AWS using these [instructions](https://www.percona.com/blog/dbaas-kubernetes-in-under-20-min/).
```
Comment (shoffman-percona) [inline docs/dbaas/setting-up.md:6]:
```suggestion
For a Kubernetes cluster that doesn't expire you can use our "easy script", you can find the instructions [here](https://www.percona.com/blog/dbaas-kubernetes-in-under-20-min/).  

In the sections that follow we'll try to outline the steps to create your own Kubernetes cluster in a few popular ways.  
```

Comment (shoffman-percona) [reply docs/dbaas/setting-up.md:6]: I'm also not sure the next section makes any sense now...or at least not where it is...perhaps the steps to install minikube should be in the minikube configure section

File: docs/dbaas/setting-up.md [hunk 5]
```diff
@@ -93,7 +93,7 @@ alias kubectl='minikube kubectl --'
     ```sh
     eksctl create cluster --write-kubeconfig --name=your-cluster-name --zones=us-west-2a,us-west-2b --kubeconfig <PATH_TO_KUBECONFIG>
     ```
-2. Follow the instructions on [How to add a Kubernetes cluster](../../get-started/dbaas.md#add-a-kubernetes-cluster) with kubeconfig from the previous step.
+2. Follow the instructions on [How to add a Kubernetes cluster](../dbaas/get-started.md#add-a-kubernetes-cluster) with kubeconfig from the previous step.
```
Comment (shoffman-percona) [inline docs/dbaas/setting-up.md:96]:
```suggestion
2. Copy the resulting kubeconfig and follow these instructions to [register a Kubernetes cluster to PMM](../dbaas/get-started.md#add-a-kubernetes-cluster).
```

File: docs/dbaas/setting-up.md [hunk 6]
```diff
@@ -93,7 +93,7 @@ alias kubectl='minikube kubectl --'
...
 
     !!! note alert alert-primary ""
         If possible, the connection details will show the cluster's external IP (not possible with minikube).
```
Comment (shoffman-percona) [inline docs/dbaas/setting-up.md:99]: What do you mean "if possible"?

File: mkdocs-base.yml [hunk 7]
```diff
@@ -273,6 +271,25 @@ nav:
       - details/victoria-metrics.md
       - details/clickhouse.md
       - details/glossary.md
+  - PMM/DBaaS:
+    - "Introduction": dbaas/index.md
+    - dbaas/architecture.md
```
Comment (shoffman-percona) [inline mkdocs-base.yml:276]: This should not be first, it should be last.

File: mkdocs-base.yml [hunk 8]
```diff
@@ -273,6 +271,25 @@ nav:
...
+    - "Setting up":
+      - "Setting up DBaaS": dbaas/setting-up.md#setting-up-dbaas
```
Comment (shoffman-percona) [inline mkdocs-base.yml:278]:
```suggestion
      - "Initial Setup": dbaas/setting-up.md#setting-up-dbaas
```

File: mkdocs-base.yml [hunk 9]
```diff
@@ -273,6 +271,25 @@ nav:
...
+      - "Create a Kubernetes Cluster": dbaas/setting-up.html#create-a-kubernetes-cluster
+      - "Deleting clusters ": dbaas/setting-up.html#deleting-clusters
```
Comment (shoffman-percona) [inline mkdocs-base.yml:280]:
```suggestion
      - "Deleting Kubernetes clusters ": dbaas/setting-up.html#deleting-clusters
```

File: docs/dbaas/databases.md [hunk 10]
```diff
@@ -0,0 +1,144 @@
...
+1. Find the row with the database cluster you want to delete.
+
+2. In the *Actions* column, open the <i class="uil uil-ellipsis-v"></i> menu and click *Delete*.
+
+3. Confirm the action by clicking *Proceed*, or abandon by clicking *Cancel*.
+
+    ![!](../_images/PMM_DBaaS_DB_Cluster_Delete.png)
+
+!!! danger alert alert-danger "Danger"
+    Deleting a cluster in this way also deletes any attached volumes.
```
Comment (shoffman-percona) [inline docs/dbaas/databases.md:89]:
```suggestion
    Deleting a cluster also deletes any attached volumes.  Please proceed with caution. 
```

**Review by shoffman-percona — COMMENTED** [review]
I put some corrections in here...this is a good start but it sill needs work on reorganizing a bit.  I see 3 distinct workflows:

1) New user with nothing
2) New user with partial setup
3) Existing user trying to do stuff

At this point I'd start getting our pro-tech writers engaged to help better organize the info for the workflows they see!

File: docs/dbaas/setting-up.md (hunk 6 above)
Comment (gen1us2k) [reply docs/dbaas/setting-up.md:99]: I had the same question :D I'll rephrase this

**Comment by rasika-chivate** [comment]
Can we add a QuickStart section so the user can get started with DBaaS quickly?

File: docs/dbaas/architecture.md [hunk 11]
```diff
@@ -0,0 +1,41 @@
...
+1. Expose Public REST API
+2. Provision Kubernetes cluster and install the following operators:
+  * Install OLM (Operator Lifecycle Manager)
+  * Percona Operator for MongoDB
+  * Percona Operator for MySQL
+  * DBaaS operator
+
+## Operator Lifecycle Manager (OLM)
+
+DBaaS uses [OLM](https://olm.operatorframework.io/docs/) to install and update operators and PMM installs OLM during the registration of Kubernetes cluster and installs Operator Catalog.
```
Comment (denisok) [inline docs/dbaas/architecture.md:20]:
```suggestion
DBaaS uses [OLM](https://olm.operatorframework.io/docs/) to install and update operators. PMM installs OLM and Operator Catalog during the registration of the Kubernetes cluster.
```

File: docs/dbaas/architecture.md [hunk 12]
```diff
@@ -0,0 +1,41 @@
...
+
+An Operator catalog is a repository of metadata that Operator Lifecycle Manager (OLM) can query to discover and install Operators and their dependencies on a cluster. OLM always installs Operators from the latest version of a catalog. DBaaS uses its own catalog for OLM that has the following operators:
+
+1. DBaaS-operator
```
Comment (denisok) [inline docs/dbaas/architecture.md:24]:
```suggestion
1. DBaaS operator
```

File: docs/dbaas/architecture.md [hunk 13]
```diff
@@ -0,0 +1,41 @@
...
+2. PXC operator
+3. PSMDB operator
+4. Victoria Metrics operator
+
+![!](../_images/dbaas_catalog.jpg)
+
+Catalog also gives the benefits of using CatalogSource and ClusterServiceVersion. ClusterServiceVersion (CSV) can solve the issue by getting the current version of installed operators via OLM API.
```
Comment (denisok) [inline docs/dbaas/architecture.md:31]: not sure what you want to say here - needs re-phrasing

**Review by denisok — APPROVED** [review]

**Review by denisok — APPROVED** [review]

**Review by recharte — COMMENTED** [review]
Great improvement :clap: 
Left some suggestions, feel free to discuss them if needed.

**Review by recharte — APPROVED** [review]

**Review by percona-csalguero — DISMISSED** [review]

**Pull request merged**